demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -0,0 +1,48 @@
+import subprocess
+import os
+from demisto_sdk.commands.common.tools import print_error, print_warning
+
+
+class ReadMeValidator:
+    def __init__(self, file_path):
+        self.file_path = file_path
+        self.is_valid = True
+        self.ready = False
+
+    def is_valid_file(self):
+        """Check whether the readme file is valid or not
+        """
+        is_readme_valid = all([
+            self.is_mdx_file(),
+        ])
+        return is_readme_valid
+
+    def is_mdx_file(self) -> bool:
+        mdx_parse = f'{os.path.dirname(os.path.abspath(__file__))}/../mdx-parse.js'
+        self.are_modules_installed_for_verify()
+        if self.ready:
```

Comment:
I dont see a use for this field. from what I understand this can be the return value of `are_modules_installed_for_verify()`

Reply:
Indeed this function should be a return value and not a field